sdk/src/gateway/strategy.ts

```diff
@@ -1,8 +1,198 @@
 import { createPublicClient, http, type PublicClient, type Chain, erc20Abi, Address } from 'viem';
-import { tokenToStrategyTypeMap } from './tokens';
-import type { StrategyAssetState, Token } from './types';
+import { ADDRESS_LOOKUP, tokenToStrategyTypeMap } from './tokens';
+import {
+    ChainId,
+    type DefiLlamaPool,
+    type EnrichedToken,
+    type PointsIncentive,
+    type StrategyAssetState,
+    type Token,
+} from './types';
 import { aaveV2AtokenAbi, compoundV2CTokenAbi } from './abi';
 
+const projectPointsIncentives: Map<string, PointsIncentive[]> = new Map([
+    [
+        'veda',
+        [
+            {
+                id: 'veda',
+                name: 'Veda Points',
+            },
+        ],
+    ],
+    [
+        'none',
+        [
+            {
+                id: 'bob',
+                name: 'Bob Spice',
+            },
+        ],
+    ],
+    [
+        'segment',
+        [
+            {
+                id: 'bob',
+                name: 'Bob Spice',
+            },
+            {
+                id: 'segment',
+                name: 'Segment Finance Points',
+            },
+        ],
+    ],
+    [
+        'ionic',
+        [
+            {
+                id: 'bob',
+                name: 'Bob Spice',
+            },
+        ],
+    ],
+    [
+        'avalon',
+        [
+            {
+                id: 'bob',
+                name: 'Bob Spice',
+            },
+            {
+                id: 'avalon',
+                name: 'Avalon Points',
+            },
+        ],
+    ],
+]);
+
+const tokensPointsIncentives: Map<string, PointsIncentive[]> = new Map([
+    [
+        // uniBTC
+        '0x236f8c0a61da474db21b693fb2ea7aab0c803894',
+        [
+            {
+                id: 'bedrock',
+                name: 'Bedrock Diamonds',
+            },
+            {
+                id: 'babylon',
+                name: 'Babylon Points',
+            },
+        ],
+    ],
+    [
+        // Solv (xSolvBTC)
+        '0x0bef2a8b771e37763c1ce02a88f404c6b2573843',
+        [
+            {
+                id: 'solv',
+                name: 'Solv XP',
```

Comment:
Does it make sense to move the name out of this mapping and having a lookup, since we will have the same name referenced multiple times?